src/providers/vod/vod-provider.ts

```diff
@@ -13,12 +13,14 @@ import {HotspotLoader} from './hotspot-loader';
 import {ThumbUrlLoader} from '../common/thumb-url-loader';
 import {CaptionLoader} from './caption-loader';
 
+const TIME_UPDATE_DELTA: number = 0.4;
```

Comment:
i think its best to save as millis just to be consistent with how we save this kind of values in other places

Reply:
yeah, but its just a matter of multiplying by 1000 then.
again, it's not crucial, it's just to be consistent with other places where we save as milliseconds.